Extended Project 3/about.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>{Codeable} - Werewolf:</title>
+    <link rel="stylesheet" href="./styles/styles.css" />
+  </head>
+
+  <body>
+    <div class="limit">
+      <header>
+        <p class="name">
+          <a class="cleanLink" href="https://codeable.pe/" target="_blank"
+            ><span class="codeable1"
+              >{code<span class="codeable2">able</span>}</span
+            ></a
+          >
+        </p>
+      </header>
+      <main>
+        <img src="YYYYYYYY7" alt="Imagen" />
+        <h1>Werewolf: YYYYYYYY8</h1>
+      </main>
+      <div class="container">
+        <div class="boxed">
+          <ul class="questionList">
+            <li>
+              <p class="question">
+                What are some honest, unfiltered things about you?
+              </p>
+              <p class="answer">YYYYYYYY1</p>
+            </li>
+
+            <li>
+              <p class="question">What drives you nuts?</p>
+              <p class="answer">YYYYYYYY2</p>
+            </li>
+
+            <li>
+              <p class="question">
+                How can people earn an extra gold star with you?
+              </p>
+              <p class="answer">YYYYYYYY3</p>
+            </li>
+
+            <li>
+              <p class="question">
+                What are some things that people might misunderstand about you
+                that you should clarify?
+              </p>
+              <p class="answer">YYYYYYYY4</p>
+            </li>
+
+            <li>
+              <p class="question">
+                What's the best way to convince you to do something?
+              </p>
+              <p class="answer">YYYYYYYY5</p>
+            </li>
+            <li>
+              <p class="question">How do you like to get feedback?</p>
+              <p class="answer">YYYYYYYY6</p>
```

Comment:
What does YYYYY[number] mean?

Reply:
It is a format that we use to replace for another word when we read all file as text. You can see this in the file "Logic.rb" line 29